src/Microsoft.Azure.ServiceBus/MessagingEventSource.cs

```diff
@@ -1150,6 +1150,21 @@ void ExceptionReceivedHandlerThrewException(string exception)
             WriteEvent(99, exception);
         }
 
+        [NonEvent]
+        public void LinkStateLost(string clientId, string receiveLinkName, AmqpObjectState receiveLinkState, bool isSessionReceiver, Exception exception)
+        {
+            if (this.IsEnabled())
+            {
+                this.LinkStateLost(clientId, receiveLinkName, receiveLinkState.ToString(), isSessionReceiver, exception.ToString());
```

Comment:
>exception.ToString() [](start = 110, length = 20)

nit: considering this is OperationCancelled exception. we will probably not have any more details in the exception trace. just logging exception.message will be sufficient here?